src/services/sanity.js

```diff
@@ -886,6 +886,39 @@ export async function fetchPackChildren(railcontentId) {
   return fetchChildren(railcontentId, 'pack');
 }
 
+export async function fetchChallengeOverview(id) {
```

Comment:
This is an ignorant question that i couldn't find a quick answer to. Is this method intended to only get `Challenge` type content? If so, I'd prefer if we moved the fields to the `contentTypeConfig.js` to maintain the design.